core/src/com/unciv/models/stats/Stats.kt

```diff
@@ -118,6 +118,16 @@ open class Stats(
         happiness *= number
         faith *= number
     }
+    
+    operator fun div(number: Float) = Stats(
```

Comment:
Nice!

Reply:
Hm I wonder, we could implement this as simply "times(1/number)", performance difference would be basically non existent

Reply:
Probably better for future changability

Reply:
We just optimized the hell out of Stats, and Xander just followed the pattern. I agree gain is negligible, simpler has precedence. We could just have gone with "[+0.5 Stat] for every global follower..." and avoided the div :wink: